boards/ikea-tradfri/include/board.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_ikea-tradfri IKEA TRÅDFRI modules
+ * @ingroup     boards
+ * @brief       Support for the IKEA TRÅDFRI modules
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the IKEA TRÅDFRI modules
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+
+#include "periph_conf.h"
+#include "periph/gpio.h"
+#include "periph/spi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Xtimer configuration
+ *
+ * The timer runs at 250 KHz to increase accuracy.
+ * @{
+ */
+#define XTIMER_HZ           (250000UL)
+#define XTIMER_WIDTH        (16)
+/** @} */
+
+/**
+ * @name    LED pin definitions
+ * @{
+ */
+#define LED0_PIN            GPIO_PIN(PA, 1)
+#define LED1_PIN            GPIO_PIN(PB, 13)
```

Comment:
From your video I take that this pin is connected to the FET switching the actual light, right? So what is LED0 connected to?

Reply:
I have to think about this, and see what is useful. In my case, PA1 was also soldered, but I don't know what it does (haven't traced it properly).

Since the same board is used in different light bulbs or accessories, I had this idea of defining different pins on an additional compiler flag. E.g. `IKEA_PRODUCT=ligthbulb` or something like that.

Reply:
I see. So I guess lets leave it for now and see what further reverse engineering brings...

Reply:
After closer inspection, it seems that PA1 is only connected for fixing the PCB. It leads to a dead end (missing resistor).

Reply:
Maybe it's connected in the more expensive versions? The versions that can change the white color between warm and cold (not only brightness) have 2 different LED-types for this - cold whites and warm whites. So there's probably one color on each connector?

Reply:
That is definitely possible. In other teardowns you can see that more pins are connected.